start-workspace-from-url.adoc

```diff
@@ -0,0 +1,85 @@
+:prod-short: Che
+
+= Starting a workspace from an URL
+
+After using the getting started examples from the Dashboard, users will want to create workspaces from existing Git repositories.
+Create workspaces from a Git repository by entering a custom URL on the {prod-short} dashboard page.
+
+== Short URL syntax
+
+Historically, the URL syntax was starting with `/f/?url=` like `https://my-che-server/f/?url=https://github.com/my/repository` but it was complex and error prone. While this syntax is still supported, short syntax should be used.
```

Comment:
```suggestion
In the earlier versions of Che, the {prod-short} URL syntax began with `/f/?url=`, for example `https://my-che-server/f/?url=https://github.com/my/repository`, but it was complex and error prone. While that syntax is still supported, short syntax should be used.
```

Reply:
we should use {prod-short} instead of  `Che`  no ?

Reply:
I've edited your suggestion to replace Che by {prod-short}